Formula/gradle.rb

```diff
@@ -1,8 +1,8 @@
 class Gradle < Formula
   desc "Build system based on the Groovy language"
   homepage "https://www.gradle.org/"
-  url "https://services.gradle.org/distributions/gradle-4.0.1-all.zip"
-  sha256 "8a8005633be0ca38a206f2590445685683452a0b98a5d8ffd5b8413be09bf998"
+  url "https://services.gradle.org/distributions/gradle-4.0.2-bin.zip"
```

Comment:
Unfortunately, this change isn't sufficient. The sha256 hash is different between the two versions of the binary.